proto-lens-tests/package.yaml

```diff
@@ -132,6 +132,8 @@ tests:
       - -Werror
     dependencies:
       - proto-lens-tests
+      # For MIN_VERSION_ghc.
```

Comment:
Remove now? (I forgot it in the commit I pushed)

Reply:
Removed